### DISCUSSION TITLE
Finding downloads / Help

### DISCUSSION THREAD

**Discussion by istrait** [haugene/docker-transmission-openvpn#2090] [Q&A] [answered]
Sorry, NOOB here trying to set this up on a Synology NAS running DSM 7.0 with docker.

It works, it is downloading files.  I just do not know where.  Cannot find them.

I have attached the docker settings file and here is a picture of my volume settings.
![Screenshot 2021-11-30 230049](https://user-images.githubusercontent.com/29987127/144187149-65fdea02-8c9c-417e-ba3e-e0a6f93ac381.jpg)

What other information can I give to help you help me find the files?

**Comment by pkishino** [comment]
suggestion: don't use the synology docker gui..it is broken in several places, you can find details in the issues etc..
I recommend using portainer..
anyway,
te screenshot shows the locations
File/folder is your location
/docker/transmission-openvpn something..

**Reply by istrait** [reply]
Will try this.  Once I figure out how to use it.  Does anyone else have any suggestions?

**Comment by istrait** [comment, marked as answer]
I figured it out.  I had the wrong directoried in the docker env settings.
